src/less/lightbox-dialog/lightbox-dialog.less

```diff
@@ -25,6 +25,40 @@
     .dialog-footer-content();
 }
 
+.lightbox-dialog__image {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: var(--border-radius-100) var(--border-radius-100) 0 0;
+    height: 218px;
+    position: absolute;
+    width: 100%;
+}
+
+.lightbox-dialog__window--expressive {
+    padding-bottom: @spacing-100;
+}
+
+.lightbox-dialog__window--expressive .lightbox-dialog__header > * {
+    margin-top: 218px;
+}
+
+.lightbox-dialog__window--expressive .lightbox-dialog__header {
+    margin: @spacing-300 @spacing-300 0;
+}
+
+.lightbox-dialog__window--expressive .lightbox-dialog__main,
+.lightbox-dialog__window--expressive .lightbox-dialog__footer {
+    padding: @spacing-200 @spacing-300;
+}
+
+.lightbox-dialog__window--expressive
```

Comment:
I thought we were going to set the modifier on the top-level element. Not sure we need to go in one level into the window. Maybe I went with the inertia of what was already implemented to suggest other subsequent changes, but this is what I was originally recommending we do:

`.lightbox-dialog--expressive` instead of `.lightbox-dialog__window--expressive`.